TheMovieDb/Controllers/ListViewController.swift

```diff
@@ -7,42 +7,67 @@
 
 import UIKit
 
-class ListViewController: UICollectionViewController {
+class ListViewController: UIViewController {
 
-    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
+    lazy private var activityIndicator: UIActivityIndicatorView = {
+        let view = UIActivityIndicatorView(style: UIActivityIndicatorView.Style.large)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.color = .systemIndigo
+        return view
+    }()
+    
+    lazy private var collectionView: UICollectionView = {
+        let layout = ColumnFlowLayout()
+        layout.scrollDirection = .vertical
+        let collection = UICollectionView(frame: CGRect(x: 0, y: 0, width: 0, height: 0), collectionViewLayout: layout)
```

Comment:
There is a special property of `CGRect` for this cases that the frame is zero. It is called `CGRect.zero` and you could have used it like this: 

`UICollectionView(frame: .zero, collectionViewLayout: layout)`

Reply:
You're right, will change it